code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -565,6 +572,13 @@ GLOBAL_VAR(bomb_set)
 		STOP_PROCESSING(SSobj, src)
 		return ..()
 
+	if(!restricted_to_station) // Non-restricted NADs should be allowed to be deleted, otherwise it becomes a restricted NAD when teleported
+		message_admins("[src] (unrestricted) has been deleted in ([diskturf ? "[diskturf.x], [diskturf.y] ,[diskturf.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[diskturf.x];Y=[diskturf.y];Z=[diskturf.z]'>JMP</a>":"nonexistent location"]). It will not respawn.")
+		log_game("[src] (unrestricted) has been deleted in ([diskturf ? "[diskturf.x], [diskturf.y] ,[diskturf.z]":"nonexistent location"]). It will not respawn.")
+		GLOB.poi_list.Remove(src)
+		STOP_PROCESSING(SSobj, src)
```

Comment:
why call `STOP_PROCESSING(SSobj, src)` here if a non-restricted NAD will runtime if it processing before it ever reaches this?

Reply:
I'd just rather be safe than sorry. If you think its unneeded I can remove it.

Reply:
Safety is good, I think this is fine to keep